app/src/main/java/camp/nextstep/edu/calculator/CalculatorContract.kt

```diff
@@ -0,0 +1,16 @@
+package camp.nextstep.edu.calculator
+
+interface CalculatorContract {
+    interface View {
+        var presenter: Presenter
```

Comment:
```suggestion
        val presenter: Presenter
```
인터페이스에서는 val로 정의해도, var로 오버라이딩이 가능해요.
이렇게 하게되면, 값의 변경이 필요한 곳에서는 var로, 값의 변경이 필요 없는 경우에는 val로 사용할 수 있어요.

Reply:
그렇군요!
```kotlin
  override lateinit var presenter: CalculatorContract.Presenter 
```
이렇게 사용하는 쪽에서 오바라이딩 해서 사용할 수 있던 거네용~ 리뷰 감사드려요~